core/resources.rs

```diff
@@ -77,4 +77,5 @@ impl ResourceTable {
 /// that allows to cast resource to concrete type in `TableResource::get`
 /// and `TableResource::get_mut` methods.
 pub trait Resource: Downcast + Any + Send {}
+impl<T> Resource for T where T: Downcast + Any + Send {}
```

Comment:
this is odd

Reply:
why?

Reply:
I don't understand what it's for

Reply:
To get rid of this useless wrapper: https://github.com/denoland/deno/pull/3903/files#diff-ad459c7e75ed9915128480db368e6c98L18.

Reply:
Look at the commit in isolation: https://github.com/denoland/deno/pull/3903/commits/a85408a18ae4dced85f61bc8ce1d49209a0302e.